posthog/src/main/java/com/posthog/internal/PostHogRemoteConfig.kt

```diff
@@ -331,6 +359,29 @@ internal class PostHogRemoteConfig(
         return parsedPayloads
     }
 
+    private fun normalizeDecideResponse(decideResponse: PostHogDecideResponse): PostHogDecideResponse {
+        val flags = decideResponse.flags
+        config.logger.log("Feature flags: $flags")
```

Comment:
does that help? not sure if this stringifies correctly or just print out the class type

Reply:
Ha nope! That was an attempt to add some debug output while looking into a bug. Not only did in not work, I forgot to clean it up. Nice catch!